content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -68,7 +68,8 @@ For a reference to old feature gates that are removed, please refer to
 | `APIServerIdentity` | `true` | Beta | 1.26 | |
 | `APIServerTracing` | `false` | Alpha | 1.22 | 1.26 |
 | `APIServerTracing` | `true` | Beta | 1.27 | |
-| `AdmissionWebhookMatchConditions` | `false` | Alpha | 1.27 | |
+| `AdmissionWebhookMatchConditions` | `false` | Alpha | 1.27 | 1.27 |
```

Comment:
Extra column of `1.27`?

Reply:
The last element is for the `until` column - just stating that the feature was alpha in 1.27